f2/src/i18n.config.js

```diff
@@ -13,6 +13,11 @@ export async function activate(locale) {
     catalog = await import(
       /* webpackChunkName: "i18n-[index]" */ `@lingui/loader!./locales/${locale}.json`
     )
+
+    const title = locale === 'fr' ? 'signaler une fraude' : 'Report a scam'
```

Comment:
Hi Justin,

LGTM, 
But what do you think of useEffect? To do it in Toggler component of LocaleSwticher.js.

Reply:
Good idea, I will test and update if everything works.

Reply:
Tested useEffect, it works when used in the LocaleSwitcher component. Toggler has an inverted locale prop to display the other language - Français when in English, and English when in French.